### ISSUE TITLE
Performance on the REDS dataset introduced in Spk2ImgNet.

### ISSUE THREAD

**Issue by chenkang455** [GitCVfb/STIR#1]
Hi, @GitCVfb, thank you for your excellent work! I am currently trying to train your STIR method on the REDS dataset from Spk2ImgNet. However, I've noticed that the performance is relatively low, achieving approximately 37.52 dB in PSNR, whereas other methods like Spk2ImgNet , WGSE  and retrained BSF reach 38+ dB. Since the dataset configuration differs, I have adapted the training configuration from WGSE to train STIR. Below are the modifications I made:

1. Input dimensions:
```python
if self.spike_dim == 61:
    block1 = x[:, 0 : 21, :, :]
    block2 = x[:, 20 : 41, :, :]
    block3 = x[:, 40 : 61, :, :]
elif self.spike_dim == 41:
    block1 = x[:, 0 : 15, :, :]
    block2 = x[:, 13 : 28, :, :]
    block3 = x[:, 26 : 41, :, :]
```
```python
if spike_dim == 61:
    self.resnet = ResidualBlock(in_channels=21, num_channels=11, use_1x1conv=True)
    dim_tfp = 16 # 5 + num_channels
elif spike_dim == 41:
    self.resnet = ResidualBlock(in_channels=15, num_channels=11, use_1x1conv=True)
    dim_tfp = 15  # 4 + num_channels
```

2. Loss function: I switched to using only `L1` loss, omitting the `perceptual` loss and `pyramid` loss.

3. Training configuration, consistent with [WGSE](https://github.com/Leozhangjiyuan/WGSE-SpikeCamera/blob/main/train.py):
``` python
Dataset with roisize=128,
epochs = 700,
bs_train = 8,
optimizer = Adam(lr = 1e-4),
scheduler = MultiStepLR(milestones=[400, 600], gamma=0.2)
```
Could you advise on what further modifications are necessary to enhance the performance on the REDS dataset in Spk2ImgNet? Thank you!

Our training script comes from https://github.com/chenkang455/Spike-Zoo/blob/main/examples/train_reds_base/train_stir.py

**Comment by IannoIITR** [comment]
How were you running the eval script? I'm getting all metric values as 0.00

**Comment by chenkang455** [comment]
> How were you running the eval script? I'm getting all metric values as 0.00

How do you evaluate? I reproduce the results based on the spikezoo.

**Comment by IannoIITR** [comment]
I ran the evaluation script provided in this repository. In spikezoo which model is performing the best?

**Comment by chenkang455** [comment]
> I ran the evaluation script provided in this repository. In spikezoo which model is performing the best?

You can check the results here: https://spike-zoo.readthedocs.io/zh-cn/latest/%E5%8F%91%E8%A1%8C%E7%89%88%E6%9C%AC%E4%BB%8B%E7%BB%8D.html. Among them, BSF performs the best.
